### PR TITLE
chore: facilitate skipping releases

### PR DESCRIPTION
**Pull request by acazacu**
Add a rule that facilitates skipping releases.

### CODE REVIEW

**Comment by mad-it** [comment]
How will this work?

File: .github/workflows/continuous-delivery.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@ jobs:
               { "release": "patch", "type": "ci" },
               { "release": "patch", "type": "improvement" },
               { "release": "patch", "type": "refactor" }
+              { "release": false, "scope": "skip-release" },
```
Comment (oleg-koval) [inline .github/workflows/continuous-delivery.yaml:52]: How this scope is handled by action?

**Comment by acazacu** [comment]
> How will this work?

@mad-it Commit anything that has a scope of `skip-release`. Example:
```
build(skip-release): reformat some file
```

File: .github/workflows/continuous-delivery.yaml (hunk 1 above)
Comment (acazacu) [reply .github/workflows/continuous-delivery.yaml:52]: @oleg-koval https://github.com/ridedott/release-me-action/pull/233#issuecomment-652842071

**Pull request merged**